### PR TITLE
Adding kim-convergence package

### PR DESCRIPTION
**Pull request by yafshar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/kim_convergence/LICENSE [hunk 1]
```diff
@@ -0,0 +1,502 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```
Comment (dopplershift) [inline recipes/kim_convergence/LICENSE:1]: This file is already included in the 0.0.2 tarball from PyPI, so you should not include that file manually in the recipe.

File: recipes/kim_convergence/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - scipy
+    - joblib
+
+test:
+  imports:
+    - kim_edn
+    - numpy
+    - scipy
+    - joblib
+    - kim_convergence
```
Comment (dopplershift) [inline recipes/kim_convergence/meta.yaml:34]: You should only test importing the package itself (and any subpackages), not its dependencies:
```suggestion
    - kim_convergence
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
The recipe is in pretty good shape, just a couple minor things to clean up.

File: recipes/kim_convergence/LICENSE (hunk 1 above)
Comment (yafshar) [reply recipes/kim_convergence/LICENSE:1]: @dopplershift thanks, I removed the file.

File: recipes/kim_convergence/meta.yaml (hunk 2 above)
Comment (yafshar) [reply recipes/kim_convergence/meta.yaml]: @dopplershift thanks, done.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**